src/sockets/commands/admin/index.ts

```diff
@@ -5,12 +5,13 @@ import { aresetpassword } from './aresetpassword';
 import { Commands } from '../types';
 import { atestgame } from './atestgame';
 import { acreatetestaccounts } from './acreatetestaccounts';
-
+import { areset } from './areset';
 export const adminCommands: Commands = {
```

Comment:
Add a new line as well please

Reply:
Fixed it, with the re-submitted about profile.ejs.